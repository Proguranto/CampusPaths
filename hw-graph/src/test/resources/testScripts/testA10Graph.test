# Tests graph creation/viewing with an acyclic graph with
# 1 node and 0 edges.

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1