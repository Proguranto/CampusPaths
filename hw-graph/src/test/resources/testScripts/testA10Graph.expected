# Tests graph creation/viewing with an acyclic graph with
# 1 node and 0 edges.

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1
the children of n1 in graph1 are: